app/views/shared/views/_show_containers_dashboard.html.haml

```diff
@@ -0,0 +1,125 @@
+.container-fluid.container-tiles-pf.containers-dashboard.miq-dashboard-view{"ng-app"        => "containerDashboard",
```

Comment:
Either way, an alternative would be to make the partial only from line 9 on ..but yeah, that has it's own problems :)